### PR TITLE
Update release to build without cleveldb

### PR DESCRIPTION
**Pull request by Taztingo**
## Description

Updates linux release to build without cleveldb.

closes: #1782

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit and integration [tests](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -51,6 +51,9 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * Create a default market in `make run`, `localnet`, `devnet` and the `provenanced testnet` command [#1757](https://github.com/provenance-io/provenance/issues/1757).
 * Remove the rust upgrade handlers [PR 1774](https://github.com/provenance-io/provenance/pull/1774).
 
+### Bug Fixes
+* Update linux release to not build with CLevelDB [PR 1783](https://github.com/provenance-io/provenance/pull/1783).
```
Comment (SpicyLemon) [inline CHANGELOG.md:55]:
```suggestion
### Bug Fixes

* Update linux release to not build with CLevelDB [PR 1783](https://github.com/provenance-io/provenance/pull/1783).
```

**Review by iramiller — DISMISSED** [review]
I think bug fix is fine for this ... mostly because we didn't intend for CLevelDB to be a default option anymore.

**Review by iramiller — DISMISSED** [review]

**Review by iramiller — APPROVED** [review]

**Review by nullpointer0x00 — APPROVED** [review]

**Comment by SpicyLemon** [comment]
I don't think we want to do this until cleveldb is no longer an option (from the SDK).

**Comment by iramiller** [comment]
> I don't think we want to do this until cleveldb is no longer an option (from the SDK).
> 
> We've added the warnings, but we want a major release to go out with those warning before we actually remove it from the build.

We should merge this after the current 1.18 goes out with the warnings

**Comment by Taztingo** [comment]
This is smaller part of #1862.

**Pull request closed without merging**